config/example.conf

```diff
@@ -8,3 +8,4 @@ initialisms = ["ACL", "API", "ASCII", "CPU", "CSS", "DNS",
 	"XSS", "SIP", "RTP"]
 dot_import_whitelist = []
 http_status_code_whitelist = ["200", "400", "404", "500"]
+error_functions = ["errors.New", "fmt.Errorf"]
```

Comment:
All checks that make use of this new option need tests, making sure to test edge cases.

Reply:
If anything, it doesn't break the old behavior.

what is the best way to make tests for this?

Reply:
Check `stylecheck/testdata/src`. Each check has a directory of test cases.

Reply:
I will try to write tests for this.